CardIO.podspec

```diff
@@ -0,0 +1,50 @@
+Pod::Spec.new do |spec|
+    spec.name = 'CardIO'
+    spec.version = '1.0.0'
+    spec.summary = 'Revolut - CardIO'
+    spec.description = 'Revolut - CardIO'
+    spec.homepage = 'https://bitbucket.org/revolut/revolut-ios'
+    spec.source = { :git =>'https://bitbucket.org/revolut/revolut-ios' }
+    spec.license = { :type => 'Proprietary' }
+    spec.author = { 'Revolut' => 'team@revolut.com' }
```

Comment:
What do you guys think I should put in `email`?

Reply:
`ios@revolut.com` perhaps? Do you know how it can be used? Just in case someone sees that and wants to contact us?

Reply:
Yep, just to contact us. I can either put `none@none.com` or a real one. No preference on my side 😄

Reply:
Okey, I am going with `none@none.com` since we dont look at emails. They can always open an issue or pr in github if needed